data/modaction

```diff
@@ -0,0 +1,47 @@
+Building
+Mining
+Fishing
+Hoarding
+Digging
+Arching
+Grinding
+Suffering
+Eating
+Fleeing
+Flying
+Dying
+Falling
+Burning
+Frolicking
+Hammering
+Slashing
+Gambling
+Helping
+Hurting
+Stealing
+Cheating
+Sleeping
+Killing
+Smashing
+Punching
+Exploding
+Poisoning
+Trolling
+Snoring
+Derping
+Pillaring
+Digging straight down
+Hugging a creeper
+Standing
+Sitting
+Shitposting
+Blogging
+Tweeting
+Snacking
+Torturing
+Lawnmowing
+Cultivating
+Studying
+Parading around
+Exploring
+Painting
```

Comment:
should these all be capitalized?

Reply:
Yes, they are capitalized like `modthing` and used in `modname`s.